static/app/views/replays/replays.tsx

```diff
@@ -205,13 +217,23 @@ function Replays(props: Props) {
 
 const StyledPageHeader = styled(PageHeader)`
   background-color: ${p => p.theme.surface100};
+  min-width: max-content;
   margin-top: ${space(1.5)};
   margin-left: ${space(4)};
 `;
 
 const StyledPageContent = styled(PageContent)`
+  padding: ${space(1.5)} ${space(2)};
   box-shadow: 0px 0px 1px ${p => p.theme.gray200};
-  background-color: ${p => p.theme.white};
+  background-color: ${p => p.theme.background};
+`;
+
+const StyledPanelTable = styled(PanelTable)`
+  grid-template-columns: minmax(0, 1fr) max-content max-content;
+
+  @media (max-width: ${p => p.theme.breakpoints[0]}) {
```

Comment:
Coupling this to the `isScreenLarge` variable, or `useMedia()` seems like it might be helpful to the next person making changes, but I can't sort out a suggestion for how.

Reply:
Oh I saw that used somewhere else where a bool prop was passed into the styled component. I bet I can just pass the same media query string in, lemme see here.

Reply:
It could work like:
`<StyledPanelTable isScreenLarge>`

```
const StyledPanelTable = styled(PanelTable)<{isScreenLarge: boolean}>`
  grid-template-columns: ${
    p => p.isScreenLarge
      ? 'minmax(0, 1fr) max-content max-content'
      : 'minmax(0, 1fr) max-content'}
`;
```

but is that better? we're re-rendering anyway to drop the extra column so maybe it's not forced?

Reply:
Yeah I see what you mean, I thought you could pass the string straight in and styled component would interpret it as part of the css. But that doesn't work.

```javascript
const StyledPanelTable = styled(PanelTable)<{mediaQuery: string}>`
  grid-template-columns: minmax(0, 1fr) max-content max-content;

  @media ${p => p.mediaQuery} {
    grid-template-columns: minmax(0, 1fr) max-content;
  }
`;
```

I'm good with leaving it as-is if you are, it's pretty easy to follow what's up right now I think.